common/eip_3076/src/lib.rs

```diff
@@ -1,10 +1,15 @@
-use crate::InterchangeError;
 use serde::{Deserialize, Serialize};
 use std::cmp::max;
 use std::collections::{HashMap, HashSet};
+#[cfg(feature = "json")]
 use std::io;
 use types::{Epoch, Hash256, PublicKeyBytes, Slot};
```

Comment:
These `types` imports add a lot of bloat to this crate. It doesn't really matter much right now since `eth2` needs `types` anyway but it might be worth considering some kind of `core` or `primitives` feature in `types` which only imports these basic types, especially if this crate gets used in  contexts where the rest of the types aren't already required.